web/assets/css/components.css

```
@@ -0,0 +1,1713 @@
+
+.overview-topology-panel .topology-canvas {
+  max-height: min(440px, 46vh);
+}
+
+.topology-stage {
+  position: relative;
+  /* 节点卡片几何的唯一定义处：.topology-lines 的左右边界由此推导，
+     改卡片宽度/边距时连线端点自动跟随，避免再出现连线穿透卡片 */
+  --topology-node-width: min(30%, 270px);
+  --topology-node-inset: 18px;
+  width: 100%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)